docs/setting-up/client/postgresql.md

```diff
@@ -309,6 +321,24 @@ where:
 - `USER`: Database user allowed to connect via TLS. Should match the common name (CN) used in the client certificate.
 - `SERVICE`: Name to give to the service within PMM.
 
+#### Automatic discovery via CLI
+
+Starting with PMM 2.41.0, there is a new flag in `pmm-admin` to limit Auto-discovery:
+
+`--auto-discovery-limit=XXX`
+
+If the limit is equal to or greater than the number of databases, Automatic-discovery is enabled. If the limit is exceeded, then Automatic-discovery is disabled.
+
+In case of a default value of zero, `Automatic- discovery` is enabled with no limit.
```

Comment:
Zero means enabled, but it is server defined. We should mention it.

There is also case with negative values. In case limit is -1 or less (like -2 etc) it is always disabled.